src/applications/static-pages/events/components/Search/index.js

```diff
@@ -1,7 +1,9 @@
 import React, { useEffect, useState } from 'react';
```

Comment:
Most changes took place in this file, where I extracted 3 components into separate files. Nothing at all should've changed related to functionality.